src/runtime.cc

```diff
@@ -96,6 +98,10 @@ namespace detail {
 		cfg = std::make_unique<config>(argc, argv, *default_logger);
 		graph_logger->set_level(cfg->get_log_level());
 
+		if(const uint32_t cores = affinity_cores_available(); cores < min_cores_needed) {
+			default_logger->warn("Too few cores available. Recomended at least: {}; available: {}", min_cores_needed, cores);
```

Comment:
Typo "Recom**m**ended". In general, can we rephrase to be a bit more descriptive? Something like
> Celerity has detected that only X logical cores are available to this process. It is recommended to assign at least 4 logical cores. Performance may be negatively impacted.